.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     continue-on-error: true
     steps:
     - name: Git Checkout
-      uses: actions/checkout@v4.1.1
```

Comment:
Actions are auto-generated and updated through zio-sbt, so I'd prefer not to edit them manually.